File: Scripts/src/icarus/model/EL/table/TGROUP/TFOOT.js
/** @module */
import TGROUP, { MODEL, TD, TH, TR } from './TGROUP.js';
/** A TABLE Footer Group */
export default class TFOOT extends TGROUP {
    /** TFOOT Constructor
        @param {CONTAINER} node Node
        @param {ContainerModel} [model] Model
    */
    constructor(node, model) {
        super(node, 'TFOOT', model);
        this.addClass('table-footer');
    }
}
export { MODEL, TD, TH, TR }

File: Models/Icarus/Elements/TFOOT.cs
using ICARUS.Models.Icarus.Elements.Attributes;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace ICARUS.Models.Icarus.Elements {

    /// <summary>
    /// Represents a table footer
    /// </summary>
    public class TFOOT : Container {

        /// <summary>
        /// An article is the basic building block for nearly all pages.  
        /// Articles are made up of a collection of SECTIONS
        /// </summary>
        /// <param name="attributes"></param>
        public TFOOT() : base("TFOOT", new MODEL() {
            label = "TFOOT"
        }) {

        }

        /// <summary>
        /// Construct a TFOOT based on the provided Form Post
        /// </summary>
        /// <param name="formPost"></param>
        public TFOOT(FormPost formPost) : base("TFOOT", formPost) {

        }
    }
}